Plugins/MesserliOneRepositoryPlugin/TargetFrameworkProvider.cs

```diff
@@ -10,131 +10,126 @@
+            ? from year in dateParts[0].ParseInt32OrNone()
+            from month in dateParts[1].ParseInt32OrNone()
+            from day in dateParts[2].ParseInt32OrNone()
+            select new DateTime(year, month, day)
+            : Option<DateTime>.None;
+    }
 
-            return installed.Contains(sdk.SdkVersion)
-                ? "Installed".Pastel(Color.Green)
-                : "Not Installed".Pastel(Color.OrangeRed);
-        }
+    private string InstalledSdk(DotNetSdk sdk)
+    {
+        var installed = GetInstalledSdks();
 
-        private List<DotNetSdk> GetSdks()
-        {
-            using var stream = _templateLoader.GetTemplateStream(DotNetSdkResource);
+        return installed.Contains(sdk.SdkVersion)
+            ? "Installed".Pastel(Color.Green)
+            : "Not Installed".Pastel(Color.OrangeRed);
+    }
 
-            var list = Option
-                .FromNullable(stream)
-                .Match(
-                    none: () => throw new Exception("no stream"),
-                    some: s => Option.FromNullable((List<DotNetSdk>?)_jsonSerializer.ReadObject(s)));
+    private List<DotNetSdk> GetSdks()
+    {
+        using var stream = _templateLoader.GetTemplateStream(DotNetSdkResource);
 
-            return list.GetOrElse(() => throw new Exception("read object failed"));
-        }
+        var list = Option
+            .FromNullable(stream)
+            .Match(
+                none: () => throw new Exception("no stream"),
+                some: s => Option.FromNullable((List<DotNetSdk>?)_jsonSerializer.ReadObject(s)));
 
-        private IEnumerable<string> GetInstalledSdks()
-        {
-            return _installedSdks ??= GetInstalledSdksFromEnvironment();
-        }
+        return list.GetOrElse(() => throw new Exception("read object failed"));
+    }
 
-        private IEnumerable<string> GetInstalledSdksFromEnvironment()
-        {
-            var dotnet = _tools.GetTool("dotnet");
+    private IEnumerable<string> GetInstalledSdks()
+    {
+        return _installedSdks ??= GetInstalledSdksFromEnvironment();
+    }
 
-            dotnet.Execute(new[] { "--list-sdks" }, "C:/");
+    private IEnumerable<string> GetInstalledSdksFromEnvironment()
+    {
+        var dotnet = _tools.GetTool("dotnet");
 
-            return ParseSdks(dotnet.StandardOutput);
-        }
+        dotnet.Execute(new[] { "--list-sdks" }, "C:/");
 
-        private HashSet<string> ParseSdks(string sdkOutput)
-        {
-            return sdkOutput
-                .Split(new[] { "\r\n", "\r", "\n" }, StringSplitOptions.None)
-                .Where(s => s.Trim().Length != 0)
-                .Select(s => s.Substring(0, s.IndexOf(' ')))
-                .ToHashSet();
-        }
+        return ParseSdks(dotnet.StandardOutput);
     }
+
+    private static HashSet<string> ParseSdks(string sdkOutput)
+        => sdkOutput
+            .Split(new[] { "\r\n", "\r", "\n" }, StringSplitOptions.None)
```

Comment:
`.SplitLines`?

Reply:
Fixed in 44c29e3ae6ba7d5c45101fa61738bc819fbb572d